src/main/java/com/erii/user/UserDetails.java

```diff
@@ -0,0 +1,105 @@
+package com.erii.user;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
+ * The UserDetails class provides methods to retrieve user details such as name, birthday, and gender.
+ */
+public class UserDetails {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final Pattern datePattern = Pattern.compile("^\\d{2}/\\d{2}/\\d{4}$");
+    private static String UserName = "";
+    private static String UserBirthday = "";
+    private static String UserGender = "";
```

Comment:
Variable names must be in camelCase.

https://se-education.org/guides/conventions/java/basic.html#naming